pkg/cmd/cli/cmd/request_project.go

```diff
@@ -16,22 +16,17 @@ import (
 	projectapi "github.com/openshift/origin/pkg/project/api"
 )
 
-type NewProjectOptions struct {
-	ProjectName string
-	DisplayName string
-	Description string
-
-	Name   string
-	Server string
+// RequestProjectCommandName is the command name used in cli.
+const RequestProjectCommandName = "new-project"
 
-	SkipConfigWrite bool
-
-	Client client.Interface
-
-	ProjectOptions *ProjectOptions
-	Out            io.Writer
-}
+// RequestProject flags constants.
+const (
+	RequestProjectDisplayNameFlag     = "display-name"
```

Comment:
I'm not 100% about moving a lot of stuff to constants. I'm ok with the other ones, but I think having the flag names in the line where they are registered makes it easier to read when traversing the code.